apps/webClient/src/views/Home/index.tsx

```diff
@@ -41,7 +44,7 @@ const Home: React.FC<Props> = ({}) => {
             </Typography>
           </Grid>
           <Grid item container justify="center">
-            <Box width="50%">
+            <Box maxWidth="75%">
```

Comment:
I expect that now this will look less good on non mobile screens.  Instead of a box here I think we should just use xs, sm sizes on the grid item because that's why we use a grid -- its easy to size the width of columns at various view port sizes.

Reply:
```<Grid item container justify="center" xs={..} sm={..} />```

if that doesn't work, then we should just make two grid elements:

```
<Grid container justify="center" spacing?>
  <Grid item xs= sm=>
    <Tyography>
```